install.py

```diff
@@ -1031,5 +1039,13 @@ def yes_or_no(value):
     return ('yes' if value else 'no')
 
 
+def search_for(path, filename):
+    for r,d,f in os.walk(path):
+        if os.path.isfile(os.path.join(r,filename)):
```

Comment:
The list of files is already in the `filenames` part, so you may save yourself a `stat()` call:
```suggestion
        if filename in filenames:
```